components/01-atoms/images/icons/_icon.twig

```diff
@@ -29,18 +30,18 @@
   {% endif %}
 
   {% if icon_title %}
-    aria-labelledby="title-{{ icon_name }}"
+    aria-labelledby="title-{{ unique_name }}"
   {% endif %}
 
   {% if icon_desc %}
-    aria-describedby="desc-{{ icon_name }}"
+    aria-describedby="desc-{{ unique_name }}"
   {% endif %}
 >
   {% if icon_title %}
-    <title id="title-{{ icon_name }}">{{ icon_title }}</title>
+    <title id="title-{{ unique_name }}">{{ icon_title }}</title>
   {% endif %}
   {% if icon_desc %}
-    <desc id="desc-{{ icon_name }}">{{ icon_desc }}</desc>
+    <desc id="desc-{{ unique_name }}">{{ icon_desc }}</desc>
```

Comment:
@ryanhagerty does this actually work here? From my brief googling last week, I think this belongs inside the `<svg` tag... and if that's the case, I'm not sure we can use it at all...

Reply:
Oh, I see we are inside the `<svg` element...

And even though [the MDN examples](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc) show the `<desc` inside an actual path element, [this discussion](https://stackoverflow.com/questions/59563251/inline-svg-title-desc-correct-usage-for-accessilibility/59563793) makes me think we're doing it okay.